tests/sundials/n_vector.cc

```diff
@@ -329,7 +329,9 @@ test_get_communicator()
 {
   auto vector   = create_test_vector<VectorType>();
   auto n_vector = make_nvector_view(vector);
-  Assert(N_VGetCommunicator(n_vector) == MPI_COMM_WORLD, NVectorTestError());
+  Assert(*static_cast<MPI_Comm *>(N_VGetCommunicator(n_vector)) ==
```

Comment:
`N_VGetCommunicator` returns a pointer?

Reply:
Yes, a `void*` to be exact. See the SUNDIALS manual:

> Returns a pointer to the MPI Comm object associated with the vector (if applicable). For
MPI-unaware vector implementations, this should return NULL.

Reply:
Thanks.